src/neo/SmartContract/StorageItem.cs

```diff
@@ -66,7 +66,7 @@ public StorageItem Clone()
 
         public void Deserialize(BinaryReader reader)
         {
-            Value = reader.ReadVarBytes();
+            Value = reader.ReadBytes((int)(reader.BaseStream.Length - reader.BaseStream.Position));
```

Comment:
Position always will be 0, isn't it?
```suggestion
            Value = reader.ReadBytes((int)(reader.BaseStream.Length));
```

Reply:
Right, fixed.